uni/lib/view/restaurant/widgets/restaurant_page_card.dart

```diff
@@ -49,9 +49,14 @@ class CardFavoriteButtonState extends State<CardFavoriteButton> {
   bool isFavorite = false;
```

Comment:
I meant init here right away! No need for `initState` since it is sync.

Reply:
I'm not getting it. Can you rephrase?

Reply:
The restaurant variable is not defined for the state instance. So I don't think is possible to reference it, which is necessary in the isFavorite definition.